game/ffmpeg.cc

```diff
@@ -24,11 +24,6 @@ extern "C" {
 #include <libavutil/error.h>
 }
 
-// Some versions of libav does not contain this definition.
-#ifndef AV_ERROR_MAX_STRING_SIZE
-#	define AV_ERROR_MAX_STRING_SIZE 64
-#endif
-
```

Comment:
current version we're using/supporting does have this or do not need this definition anymore?

Reply:
This is not my change, but merge from master, which no longer has that code.

Reply:
Ahh check